Bdd/features/environment.py

```
@@ -1,7 +1,44 @@
+    # Clean up any long-lived interactive process
+    if hasattr(context, "interactive_process"):
+        process = context.interactive_process
+        if process.poll() is None:
+            try:
+                process.stdin.write("quit\n")
+                process.stdin.flush()
+            except (BrokenPipeError, OSError):
+                pass
+            try:
+                process.wait(timeout=5)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)